CMakeLists.txt

```diff
@@ -19,21 +19,28 @@ endif ()
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 
 # options
-option(BUILD_SHARED_LIBS "build shared lib" ON)
-option(MMDEPLOY_BUILD_MODEL_OPTIMIZER "build MMDeploy's model optimizer" ON)
-option(MMDEPLOY_BUILD_SDK "build MMDeploy's SDK" OFF)
-option(MMDEPLOY_ZIP_MODEL "support sdk model in zip format" OFF)
-option(MMDEPLOY_BUILD_SDK_CSHARP_API "build MMDeployExtern.dll for C# API" OFF)
-option(MMDEPLOY_BUILD_TEST "build MMDeploy's csrc's unittest" OFF)
-set(MMDEPLOY_TARGET_DEVICES
-        "cpu" CACHE STRING "MMDeploy's target devices")
-set(MMDEPLOY_TARGET_BACKENDS "" CACHE STRING "MMDeploy's target inference engines")
-set(MMDEPLOY_CODEBASES "all"  CACHE STRING "select OpenMMLab's codebases")
+option(MMDEPLOY_SHARED_LIBS "build shared libs" ON)
```

Comment:
May change `BUILD_SHARED_LIBS` to `MMDEPLOY_SHARED_LIBS` in `MMDeployConfig.cmake.in` accordingly.